sky/shell/BUILD.gn

```diff
@@ -389,8 +388,7 @@ if (is_android) {
     ]
   }
 
-  sky_ios_sdk("shell") {
-    sdk_name = "Flutter"
+  group("shell") {
   }
```

Comment:
Hm, it's still building the Flutter framework - I can try to see where it's picking up the dependency.  Previously, it only depended upon the two copies:

```
group(target_name) {
  deps = [
    ":copy_sdk_xcode_harness",
    ":copy_user_editable_files",
  ]
}
```